src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ReportingController.kt

```diff
@@ -28,4 +29,11 @@ class ReportingController(
     reportingService.generateServiceProviderPerformanceReport(dateInterval.fromDate, dateInterval.toDate, userMapper.fromToken(authentication))
     return ResponseEntity.accepted().build()
   }
+
+  @PreAuthorize("hasRole('NDMIS_REPORTING')")
```

Comment:
i know this role doesn't exist yet - but all roles start with `ROLE_` so it would be good to have that, even if we will change the role in the future.